prdoc/pr_5067.prdoc

```diff
@@ -0,0 +1,19 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Fix region nonfungible implementation
+
+doc:
+  - audience: Runtime User
+    description: |
+      PR fixes the issue with the current implementation where minting causes
+      the region coremask to be set to `Coremask::complete` regardless of the
+      actual coremask of the region.
+
+crates: 
+- name: pallet-broker
+  bump: major
+- name: coretime-rococo-runtime
+  bump: major
+- name: coretime-westend-runtime
+  bump: major
```

Comment:
```suggestion
- name: coretime-rococo-runtime
  bump: patch
- name: coretime-westend-runtime
  bump: patch
```

I don't think this is any more than a bugfix for the runtimes

Reply:
True, since we don't expose the `issue` function.

Reply:
True, since we don't expose the `issue` function.